### ISSUE TITLE
Wayland and pipewire support

### ISSUE THREAD

**Issue by Brent001** [pelya/xserver-xsdl#156]
I'm using kde plasma. And I feel slow because it's use x11 server and also pulseaudio. Please I needed support for pipewire and Wayland.

**Comment by coreyreichle** [comment]
I don't believe an X server is capable of being a Wayland client.  The KDE wayland display manager needs to be ported to android, as it's not just a feature one can bolt onto a X server implementation.

**Comment by 1liuzhongyi** [comment]
试试 termux:x11
